augur/data/lat_longs.tsv

```diff
@@ -187,6 +187,7 @@ country	portugal	39.6945	-8.13057
 country	puerto_rico	18.2459121	-66.4164147
 country	puerto rico	18.2459121	-66.4164147
 country	qatar	25.27932	51.52245
+country reunion -21.1151 55.5364
```

Comment:
Needs to change to tabs not spaces (at least, https://raw.githubusercontent.com/nextstrain/augur/c45ec7f828b3575f07a2caee67b833fc05a0666d/augur/data/lat_longs.tsv appears to be using spaces not tabs)

Reply:
oops yes!
and I agree, not much point showing reunion sequences in France just because of politics...